port/common/omrport.c

```diff
@@ -29,6 +29,7 @@
 #include "omrport.h"
 #include "omrportpriv.h"
 #include "ut_omrport.h"
+#include "omrsl.h"
```

Comment:
I don't think this is wanted: the declaration of `omrsl_get_libraries()` should be in `omrportpriv.h` instead.